### PR TITLE
Fix button padding

### PR DESCRIPTION
**Pull request by danigo99**
Fixing back from validation defect of this ticket: https://linear.app/prismic/issue/SM-804/aauser-i-see-slices-card-redesigned-with-a-new-screenshot-button

image

### CODE REVIEW

File: packages/slice-machine/src/theme.ts [hunk 1]
```diff
@@ -324,8 +324,7 @@ const AppTheme = (): Theme =>
         lineHeight: "16px",
         pl: 2,
         pr: 2,
-        pb: "6px",
```
Comment (MarcMcIntosh) [inline packages/slice-machine/src/theme.ts:327]: Is this the only use of the secondarySmall button class?

Comment (danigo99) [reply packages/slice-machine/src/theme.ts:327]: Yep. 2 references and they're both update screenshot buttons

Comment (MarcMcIntosh) [reply packages/slice-machine/src/theme.ts:327]: Okay, Is that the only change required by the ticket?

Comment (danigo99) [reply packages/slice-machine/src/theme.ts:327]: Yeah, the button height was the only comment on the ticket from the back from validation

**Review by MarcMcIntosh — APPROVED** [review]

**Pull request merged**